pkg/lib/hooks.ts

```diff
@@ -262,10 +266,10 @@ export function useObject(create, destroy, deps, comps) {
 
     destroy_ref.current = destroy;
     useEffect(() => {
-        return () => destroy_ref.current?.(ref.current);
+        return () => { destroy_ref.current?.(ref.current!) };
```

Comment:
What's the bang needed for?

Reply:
The destroy function takes a non-null pointer, but the current value is null-typed.  The `!` is an assertion which means "I promise this isn't null".